simulator/front_end/src/front_end/region_creation/region.py

```diff
@@ -0,0 +1,22 @@
+import hashlib
+
+'''
+ This class represents a Region which contains the size of the region and 
+ finger prints contained in it.
+'''
+
+
+class Region:
+    def __init__(self, max_size: int):
+        self.max_size = max_size
+        self.current_size = 0
+        self.fingerprints = []
+        self.hash = hashlib.sha1()
+
+    def add_fingerprint(self, fingerprint: bytes, fingerprint_size: int):
```

Comment:
If I'm correct about this other comment then fingerprint_size parameter would be redundant
> I think current_chunk_size is the size of the actual data. But the fingerprint hashes will all be a uniform size. The fingerprints are what we are filling the regions up with. So I think we need to count how many bytes the fingerprint itself has.